bors.toml

```diff
@@ -0,0 +1,12 @@
+status = [
+    "Rustfmt (ubuntu-latest, stable)",
+    "Clippy (ubuntu-latest, stable)",
+    "Unit tests (ubuntu-latest, stable)",
```

Comment:
Is these correct names?

Reply:
I believe yes. I tested out naming conventions on my own fork. If you use bors + it will not touch master unless they are corret